src/status_im/contexts/wallet/home/tabs/view.cljs

```diff
@@ -10,15 +10,17 @@
 
 (defn- on-collectible-long-press
   [{:keys [preview-url collectible-details id]}]
-  (let [chain-id (get-in id [:contract-id :chain-id])
-        address  (get-in id [:contract-id :address])]
+  (let [chain-id         (get-in id [:contract-id :chain-id])
+        token-id         (:token-id id)
+        contract-address (get-in id [:contract-id :address])]
     (rf/dispatch [:show-bottom-sheet
                   {:content (fn []
                               [options-drawer/view
```

Comment:
In few places `chain-id`, `token-id` and `contract-address` are all extracted from one `id` param before passing to `options-drawer/view`. Instead we can pass just `id` and make the extraction inside `options-drawer/view` to reduce text repetition. This will also reduce surface for changes if structure of `id` will change in a future.

Reply:
thanks for the great suggestions! Will update :)